pkg/types/mocks/head_subscribed.generated.go

```diff
@@ -0,0 +1,44 @@
+// Code generated by mockery v2.20.0. DO NOT EDIT.
+
+package mocks
+
+import (
+	types "github.com/smartcontractkit/ocr2keepers/pkg/types"
+	mock "github.com/stretchr/testify/mock"
+)
+
+// HeadSubscriber is an autogenerated mock type for the HeadSubscriber type
+type HeadSubscriber struct {
+	mock.Mock
+}
+
+// HeadTicker provides a mock function with given fields:
+func (_m *HeadSubscriber) HeadTicker() chan types.BlockKey {
```

Comment:
i understand this file is generated in `types.go` but what about logics in this function? this looks like we added these logics?